### DISCUSSION TITLE
Filtering Search by 'Modified Date' (LastModifiedTime) Does Not Return Correct Result Due To Date Formatting

### DISCUSSION THREAD

**Discussion by mjbudy** [microsoft-search/pnp-modern-search#3135] [Q&A]
**Version:** 3.16.0.1

**Issue:** The date range (May 1, 2023 to May 3, 2023) that I selected for the Modified Date, does not show results that include items that was actually modified on May 3,2023. I am not sure if this is a known issue and can be fixed if we upgrade the pnp modern search version. I also want to confirm if this issue is addressed in this change -- https://github.com/microsoft-search/pnp-modern-search/issues/1601? Thank you.

**Below are other details to show the issue:**

_Selected Date Range :_ 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/139618291/64a635bb-930f-4f47-9f9f-f36607ecb62d)

_LastModifiedDate Range shown in Devtools:_
![image](https://github.com/microsoft-search/pnp-modern-search/assets/139618291/8d45fdf4-63d6-4c63-9506-4b7c5cbe2877)

_Below is my current local date and time setting:_
![image](https://github.com/microsoft-search/pnp-modern-search/assets/139618291/93387104-8a63-4380-aeb1-a6c533e31b2a)

**Comment by wobba** [comment]
We don't offer support on v3 any more. Any change you need has to be contributed by you or someone to help out. Also as we use Z time on the filter, that might be incorrect for an end-user expectation. Date and time is tricky with SharePoint as you have Z time, site settings and user browser settings. Which ones to use is tricky.
